app/src/main/java/de/danoeh/antennapod/ui/screen/playback/SleepTimerDialog.java

```diff
@@ -169,11 +288,156 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
         return builder.create();
     }
 
+    private void confirmAlwaysSleepTimerDialog() {
+        @SuppressLint("VisibleForTests")
```

Comment:
Why is this needed? This is not a test

Reply:
The setter is marked as public for testing, this is the only other place that uses it outside of the tests.
IDE complains without the annotation and I didn't know if I could remove the annotation from the originating class.